librz/core/linux_heap_jemalloc.c

```diff
@@ -58,21 +58,54 @@ static GHT GH(je_get_va_symbol)(RzCore *core, const char *path, const char *sym_
 	return vaddr;
 }
 
-static int GH(je_matched)(const char *ptr, const char *str) {
+static inline GHT GH(je_matched)(const char *ptr, const char *str) {
 	int ret = strncmp(ptr, str, strlen(str) - 1);
 	return !ret;
```

Comment:
```suggestion
static bool GH(je_matched)(const char *ptr, const char *str) {
	return !strncmp(ptr, str, strlen(str) - 1);
```
why the `-1` ?

Reply:
That is how it was  implemented initially, but ye what you say makes sense as by definition the null character is not counted